app/views/dams_objects/_admin_download.html.erb

```diff
@@ -15,17 +15,20 @@
   end
 %>
 
-<div class="etc-menu">
 
-  <%= link_to embed_glyph, "#embedLink-#{component_id}", class:etc_menu_item_class, role:'button', :data => { :toggle => 'modal' }, title:'Embed' %>
+  <% if !(current_user && current_user.work_authorizations_count > 0) %>
```

Comment:
This check is performed here, as well as in the `data_viewer` partial. I wonder if we could make this a little helper method, something like `vrr_user?` or something?

Reply:
Do you mean to just add this one line conditional statement to helper method? Or do you mean to refactor the old code to the helper method? If it it the latter, i think it's better to create another refactoring ticket so that it will not block this new release.

Reply:
the `vrr_user?` sounds good.